dynamic-simulation/dynamic-simulation-dsl/src/main/java/com/powsybl/dynamicsimulation/groovy/DynamicModelExtension.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.powsybl.dynamicsimulation.groovy;
+
+import com.powsybl.dynamicsimulation.DynamicModel;
+
+/**
+ * @author Marcos de Miguel <demiguelm at aia.es>
+ */
+public interface DynamicModelExtension extends GroovyExtension<DynamicModel> {
+}
```

Comment:
To be consistent with the curves this class should be named DynamicModelGroovyExtension

Reply:
Done